packages/terra-content-container/lib/ContentContainer.scss

```diff
@@ -0,0 +1,37 @@
+.terra-ContentContainer {
+  display: flex;
+  flex-direction: column;
+
+  > .terra-ContentContainer-header {
```

Comment:
We use the descendant selectors, because of the possibility of nested content containers.